docs/includes/generated_docs/schemas/beta.tpp.md

```diff
@@ -2266,7 +2287,7 @@ TODO
     <code>ICD-10 code</code>
   </dt>
   <dd markdown="block">
-TODO
+The international classification of diseases (ICD) code used to identify the secondary patient diagnosis.
```

Comment:
```suggestion
The international classification of diseases (ICD) code used to identify the secondary patient diagnosis. NB this will typically not be completed.
```